dotnet/samples/Demos/CopilotAgentPlugins/CopilotAgentPluginsDemoSample/Plugins/CopilotAgentPlugins/AstronomyPlugin/astronomy-apiplugin.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I believe the loading path should be updated to use those instead.
https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/Concepts/Resources/Plugins/CopilotAgentPlugins

This way we don't end up with mostly duplicated content